app/enterprise/2.1.x/brain-immunity/model-training

```diff
@@ -0,0 +1,106 @@
+
+```bash
+curl -d '{"start":"2019-01-08 10:00:00", "end":"2019-01-09 23:30:00", "kong_entity":"2beff163-061d-43ad-8d87-8f40d10805ba"}' \
+ -H "Content-Type: application/json" \
+ -X POST http://<COLLECTOR_HOST>:<COLLECTOR_PORT>/trainer
+```
+
+### Checking Models Trained
+
+Only endpoints + method combinations that have a model trained can be monitored for alerts. If you want to check which endpoints have models, you can use `http://<COLLECTOR_HOST>:<COLLECTOR_PORT>/monitoredendpoints`, which return a list of all models in the system. Each item in this list contains the following identifying information for the model:
+
+* `base_url`: The URL of the traffic used to train the model.
+* `method`: The method of the traffic used to train the model.
+* `route_id`: The Kong `route_id` that the traffic used to train the model is associated with. **service_id**: The Kong `service_id` that the traffic used to train the model is associated with. **model_version_id**: The model version number of the current, active model.
+* `active_models`: A JSON object containing information on the active status of each of the six (6) core alert types in Immunity (unknown_parameters, abnormal_value, latency, traffic, status codes, and value_type).
+
+In this object, the value is a specific alert type and the value is a boolean value where `True` indicates that the model is actively monitoring for that alert type.
+
+In general, if an endpoint and method combination model does not appear on the returned object from `/monitoredendpoints`, this is likely because not enough traffic has been seen by Immunity to build a reliable model.
+
+### Configure Auto-Training
+
+#### Restarting Training Schedules
+
+Immunity automatically sets up training jobs when it first starts up, and retrains all models on an optimized schedule based on the time when data started flowing through Immunity. If you have experienced large changes in the type of data you expect to flow through Immunity and do not feel comfortable choosing an optimal time period to use for retraining with the `/trainer` endpoint, you can re-trigger Immunity's auto-training by posting to the `/trainer/reset` endpoint. Immunity will then recreate its retraining schedule as if it was just being started and newly ingesting data.
+
+```bash
+curl -X POST http://<COLLECTOR_HOST>:<COLLECTOR_PORT>/trainer/reset
+```
+
+#### Configuring Auto-Training Rules
+
+For best results, Immunity retrains on a regular basis. If you do not need to retrain your models regularly and are happy with your current model, you can stop auto retraining via a POST request to the `/trainer/config` endpoint. This endpoint takes these parameters:
+
+* `kong_entity`: The `route_id` or `service_id` for which to turn on or off auto-training.
```

Comment:
don't take kong_entity anymore. Just take workspace_name, and auto-training is turned off for the whole workspace. This might be the only endpoint where workspace_name is still accepted.